cmd/backup-manager/app/import/manager.go

```diff
@@ -186,6 +201,7 @@ func (rm *RestoreManager) performRestore(restore *v1alpha1.Restore) error {
 
 	restore.Status.TimeStarted = metav1.Time{Time: started}
 	restore.Status.TimeCompleted = metav1.Time{Time: finish}
+	restore.Status.CommitTs = commitTs
```

Comment:
always update `commitTs` when we get it successfully

Reply:
sorry, maybe it's better not to update it when the restore fails as the `commitTs` indicates the commit timestamp of the restored backup

Reply:
which means current implementation is correct?

Reply:
changes in this file LGTM

Reply:
I see